cmd/bridged/README.md

```diff
@@ -0,0 +1,24 @@
+# Bridged
+
+## Building
+
+Simply run `make` will build all commands, including bridged.
+
+## Running
+
+The prototype runs in `light mode`. The first time it is run, it will take a while for the chain to fully sync (seemingly roughly an hour).
+ 
+An account is expected to run this example. 
+- Use an existing one:
+  One can be created using the default `geth` binary, and then imported. To import an account, pass the `--account-json $ACCOUNTFILE` flag. This will import the account into the oracles keystore. You must also use the `--account-pass $ACCOUNTPASS` flag with the password which was used to encrypt the account.
+- Let the bridge create it's own:
+  If no existing one is imported or the passed accountfile has no accounts, the bridge will create an account itself, encrypted with the passed password  from the  `--account-pass $ACCOUNTPASS` flag.
+
+ After the first time, the account remains loaded (unless the keystore dir is removed/cleared), and only the password needs to be provided.
+
+**Attention** The keystore is stored per network (main, rinkeby or Ropsten testnets)
+
+### Important
+
+If you want to create these mint transactions yourself, the provided contract will need to be deployed by the account of which you have imported the key.
+The contract address can be changed [in the source](./bridge.go)
```

Comment:
Makes sense. We should point out then, that using an incorrect token contract can cause weird bugs, if there is an ABI mismatch

Reply:
I ran into an issue with an outdated contract the other day (upgraded bindings, but did not yet switch the proxy to the new contract), and it gives errors when the methods are executed (so no checks are done when loading). So this would have to be manually implemented I suppose. Also keep in mind that we use a proxy setup, so the actual contract address (which is the proxy), will have a completely different ABI then the binding. (though we can get the address of the implementation contract and then get its ABI). All in all I'd just add the flag for now and open an issue to implement verification at a later date

Reply:
Sounds good. It is either way an expected responsibility for the user. Verifying the use of the correct contract address (when custom) is no different from verifying you're sending tokens to the correct address.